### ISSUE TITLE
SPIFFS and SD card file browser

### ISSUE THREAD

**Issue by probonopd** [probonopd/WirelessPrinting#129]
SPIFFS and SD card file browser file browser so that we can store and manage more than one GCODE file on the SD card.

* For SD card, we could store all GCODEs ever sent to the printer as long as space permits
* For SPIFFS, we should only store GCODE which was specifically uploaded for this purpose (e.g., some simple calibration GCODE) in order to leave enough space

**Comment by probonopd** [comment]
Possibly this can be de-prioritized if we have https://github.com/probonopd/WirelessPrinting/issues/131 working.

**Comment by GMagician** [comment]
I've tryed some time ago to store a gcode into sd via repetier...unthinkable

**Comment by probonopd** [comment]
This ticket is about storing on the ESP SD card, not on the printer SD card. #131 is about storing on the printer SD card.

**Comment by GMagician** [comment]
SPIFF browser is already present (/editor) but not too much flash to store a lot of files

**Comment by probonopd** [comment]
Yes, we need a way to print from there, and we need this for SD.

**Comment by GMagician** [comment]
Print from /editor is almost impossible, it's linked in binary form I think and modify it is not as easy as it may seems
